CONTRIBUTING.md

```diff
@@ -3,13 +3,15 @@
 This project is work of [many contributors](https://github.com/bornfight/README/graphs/contributors).
```

Comment:
Is there a link somewhere to this file, how can I access it from the initial readme?
Or is the idea to click on the actual file

Reply:
this is githubs link :) https://github.com/bornfight/README/graphs/contributors
like /issues or /pulls 
It just shows a list of people that commit to the repo and fancy charts and graphs